Sprint-2/stretch/till.js

```diff
@@ -1,31 +1,39 @@
-// totalTill takes an object representing coins in a till
+  // totalTill takes an object representing coins in a till
 
-// Given an object of coins
-// When this till object is passed to totalTill
-// Then it should return the total amount in pounds
+  // Given an object of coins
+  // When this till object is passed to totalTill
+  // Then it should return the total amount in pounds
 
-function totalTill(till) {
-  let total = 0;
+  function totalTill(till) {
+    let total = 0;
 
-  for (const [coin, quantity] of Object.entries(till)) {
-    total += coin * quantity;
-  }
+    for (const [coin, quantity] of Object.entries(till)) {
+      total += +coin.replace("p", "") * quantity;
+    }
 
-  return `£${total / 100}`;
-}
+    return `£${total / 100}`;
+  }
 
-const till = {
-  "1p": 10,
-  "5p": 6,
-  "50p": 4,
-  "20p": 10,
-};
+  const till = {
+    "1p": 10,
+    "5p": 6,
+    "50p": 4,
+    "20p": 10,
+  };
 const totalAmount = totalTill(till);
 
 // a) What is the target output when totalTill is called with the till object
+  
+/* The target output of totalTill(till) is the total value of coins in pounds, properly formatted. */
 
-// b) Why do we need to use Object.entries inside the for...of loop in this function?
+  // b) Why do we need to use Object.entries inside the for...of loop in this function?
 
+/* The Object.entries() method is necessary because it converts the object into an array of key-value pairs, which can then be iterated with for...of */
+  
 // c) What does coin * quantity evaluate to inside the for...of loop?
+  
+/* Inside the loop, coin * quantity multiplies the key (e.g., "1p") with the quantity. */
+
+  // d) Write a test for this function to check it works and then fix the implementation of totalTill
 
-// d) Write a test for this function to check it works and then fix the implementation of totalTill
+  /* assert.strictEqual(totalTill({ "1p": 5, "2p": 10, "10p": 2 }), "£0.35"); */
```

Comment:
What do you expect the function to return when give given object is `{ "1p": 20, "10p", 10 }`?

Reply:
Good point 
Expected :  "£1.20
Returned:   "£1.2

Added toFixed(2) so now it is correct